opentelemetry-semantic-conventions/src/opentelemetry/semconv/v1_23_1/Cloudevents_attributes.py

```diff
@@ -0,0 +1,55 @@
+
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=too-many-lines
+
+from enum import Enum
+
+class CloudeventsAttributes:
```

Comment:
TODO: known issue https://github.com/open-telemetry/semantic-conventions/issues/599

Reply:
The conclusion is to tolerate it and do nothing.